### DISCUSSION TITLE
DKIM .private key file permissions

### DISCUSSION THREAD

**Discussion by thewizards** [PHPMailer/PHPMailer#2335] [Q&A]
Hi,
I'm trying to send mail via PHPMailer-DKIM Signature, but my mails keep reaching the spam folder, on inspecting further i found the following :
if I keep the permissions as 0600 I get 
SPF: PASS 
DKIM: 'FAIL'
DMARC: 'PASS'
and If I change the permissions to 0640 i do get DKIM 'PASS' but the mail is not sent until I set it back to 0600. 
Any help on the same ?

**Comment by Synchro** [comment]
Permissions that strict make ownership the more likely problem. It would mean that your script has to be run by the same user that owns the key. On a typical web server this is often `www-data`.

**Reply by thewizards** [reply]
My bad, I forgot to mention that this happened after changing the file group to www-data
